kv/fault_injection_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv_test
+
+import (
+	"github.com/juju/errors"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb"
+	"github.com/pingcap/tidb/domain"
+	"github.com/pingcap/tidb/kv"
+)
+
+type testFaultInjectionSuite struct{}
+
+var _ = Suite(testFaultInjectionSuite{})
+
+func (s testFaultInjectionSuite) TestFaultInjectionBasic(c *C) {
+	var cfg kv.InjectionConfig
+	err := errors.New("foo")
+	cfg.SetGetError(err)
+
+	store, _, err := newStoreWithBootstrap()
```

Comment:
Would `NewMockStorage` work here?
Function `newStoreWithBootstrap` is time consuming.